scripts/get-previous-y-stream.bats

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bats
+
+load "./test_helpers.bats"
+
+setup() {
```

Comment:
Nit: This can be `setup_file` as the value of `program` is the same for each test case in this file, and we do not change it during the tests.

Reply:
I did not think of this before your comment. Tried it and prefer to stay with `setup()`.

The problem with `setup_file` is that it requires `export`-ing variables. While it makes no harm so far, exported variable is seen by subprocesses such as `get-previous-y-stream.sh` itself. This makes the global state seen by that script in test context different than in execution outside of test context.

Since the overhead of current `setup()` is negligible, I'd prefer to stay with it unless you have strong opinion.

Reply:
No, no strong opinion here, it can stay with `setup` as the execution costs nothing.